packages/lib/src/components/internal/Toggle/Toggle.scss

```diff
@@ -0,0 +1,159 @@
+@use 'styles/mixins';
+@import 'styles/variable-generator';
+
+.adyen-checkout-toggle {
+  $component-root: &;
+  $label-padding: token(toggle-label-padding);
+
+  color: inherit;
+  cursor: pointer;
+  display: flex;
+  user-select: none;
```

Comment:
Was this just part of the copy paste?

I wonder if it's worth doing this.

Reply:
Doing what exactly? 🤔  

The CSS was copy-paste from Bento's Toggle component (with minor adjustments), since we use the same DOM structure.

Reply:
Use-select removes the possibility of copy and pasting code. It can be useful but I think it should be only used in extreme cases.

We don't do this for labels, and we don't do this for buttons so I don't think we should use it here too.

Reply:
Makes sense